docs/modules/mssql.md

```diff
@@ -14,6 +14,19 @@ Please run the following command to add the MS SQL Server module to your Go depe
 go get github.com/testcontainers/testcontainers-go/modules/mssql
 ```
 
+!!!info
+    In order to use this module, you must set the `GODEBUG=x509negativeserial=1` environment variable. See [https://github.com/microsoft/mssql-docker/issues/895](https://github.com/microsoft/mssql-docker/issues/895) for more details.
+
```

Comment:
```suggestion
!!!info
    To use this module with Go 1.23+, set `GODEBUG=x509negativeserial=1`. See the related issue in the [mssql-docker repository](https://github.com/microsoft/mssql-docker/issues/895) for details.
    
    ```shell
    # append to any existing GODEBUG flags instead of overwriting
    export GODEBUG="${GODEBUG:+$GODEBUG,}x509negativeserial=1"
    ```
```